build_scripts/build_usd.py

```diff
@@ -1878,6 +1883,11 @@ def InstallUSD(context, force, buildArgs):
 subgroup.add_argument("--no-tutorials", dest="build_tutorials", action="store_false",
                       help="Do not build tutorials")
 subgroup = group.add_mutually_exclusive_group()
+subgroup.add_argument("--vulkan", dest="build_vulkan", action="store_true",
```

Comment:
These options should go under the "Imaging and USD Imaging Options" argument group.